katalog/fluentd/fluentd-index-sighup-template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "index_patterns" : ["system-*","kubernetes-*","ingress-controller-*","audit-*"],
+  "settings": {
+    "number_of_shards": 3,
```

Comment:
I would use `number_of_shards: 1` and `number_of_replicas: 2`, our use case wouldn't benefit much from the sharding.

Reply:
Ok, changing to 1 shard and 2 replicas